backend/internal/api/handler_test.go

```diff
@@ -369,6 +369,66 @@ func TestCreate(t *testing.T) {
 			assert.Equal(t, tt.statusCode, recorder.Code)
 		})
 	}
+
+	t.Run("should return error with status code 406 when request content is corrupted", func(t *testing.T) {
+
+		//given
+		hook := test.NewGlobal()
+		img := model.Image{
+			ID:      fixedID,
+			Content: "notAnImage",
+			Time:    time.Now().Format(time.RFC3339),
+		}
+		jsonImg, err := json.Marshal(img)
+		assert.NoError(t, err)
+		req, err := http.NewRequest("POST", "/v1/images", bytes.NewBuffer(jsonImg))
+		assert.NoError(t, err)
+		req.Header.Set("Content-Type", "application/json")
+		recorder := httptest.NewRecorder()
+		dbManagerMock := mocks.DBManager{}
+		idMock := mocks.IdGenerator{}
+		idMock.On("NewID").Return(fixedID, nil)
+		eventBusMock := mocks.EventBus{}
+		testSubject := NewHandler(&dbManagerMock, &idMock, &eventBusMock)
+
+		//when
+		testSubject.Create(recorder, req)
+
+		//then
+		dbManagerMock.AssertNumberOfCalls(t, "Get", 0)
+		assert.Contains(t, hook.LastEntry().Message, "CREATE handler: content is not an image")
+		assert.Equal(t, http.StatusNotAcceptable, recorder.Code)
+	})
+
+	t.Run("should return error with status code 500 when content in unmarshaled struct is empty", func(t *testing.T) {
+
+		//given
+		hook := test.NewGlobal()
+		img := model.Image{
+			ID:      fixedID,
+			Content: "",
+			Time:    time.Now().Format(time.RFC3339),
```

Comment:
Can't you provide some specific Time object for the testing purposes? Depending on `time.now()` in the unit tests is not considered the best practice.